cdbot/cogs/cyber.py

```diff
@@ -408,7 +408,11 @@ async def elitecount(self, ctx: Context):
         """
         if ELITECOUNT_ENABLED:
             preferences = {
+                "2018": {
+                    "Attendees": Roles.Elite.VET2018.ATTENDEES
```

Comment:
I think to keep it consistent, this line should end with a `,`

Reply:
He's right you know